sys/cheri/cheri.h

```diff
@@ -114,6 +117,15 @@ extern void * __capability swap_restore_cap;
 /* Root of all sealed kernel capabilities. */
 extern void * __capability kernel_sealcap;
 
+#ifdef __CHERI_PURE_CAPABILITY__
+/*
+ * Kernel root capability
+ * has all permissions
+ * This spans the whole address-space
+ */
+extern void * __capability cheri_kall_capability;
```

Comment:
How about `kernel_root_cap`?

Reply:
That seems fine, though maybe it should be `kernel_cap` to match `userspace_cap`? Or maybe we should deviate because the userspace one would be better called `userspace_root_cap` (and similarly `userspace_sealroot_cap` or something). I assume it doesn't need sealing permission though, in which case the comment should be updated?

Reply:
I think I prefer moving to `userspace_root_cap` as the name is more explicit